db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/Optimizer.java

```diff
@@ -402,4 +402,7 @@ void considerCost(Optimizable optimizable,
 
     public void setSingleRow(boolean singleRowInRelation);
 
+    public void setForSpark(boolean forSpark);
+
+    public boolean getForSpark();
```

Comment:
Shouldn't it be `isForSpark` since it's a predicate?

Reply:
Done